pkg/kubelet/server_test.go

```diff
@@ -931,6 +936,237 @@ func TestServeExecInContainer(t *testing.T) {
 	}
 }
 
+// TODO: largely cloned from TestServeExecContainer, refactor and re-use code
+func TestServeAttachContainer(t *testing.T) {
```

Comment:
I wasn't worried about the diff getting handled; I didn't want to end up with a copy of the buggy code that #10837 fixed.